linode/manager
# M3-5579: Update Entity Detail in Storybook

## Description
Convert file to MDX and add the Kubernetes Cluster Detail

## How to test
Run Storybook (`yarn storybook`) and check `Linode` and `Kubernetes` under `Components/Entity Detail`
